evalml/tests/utils_tests/test_woodwork_utils.py

```diff
@@ -3,158 +3,79 @@
 import pytest
 import woodwork as ww
 
-from evalml.utils import (
-    _convert_numeric_dataset_pandas,
-    _convert_woodwork_types_wrapper,
-    infer_feature_types
-)
+from evalml.utils import _convert_numeric_dataset_pandas, infer_feature_types
 
 
-def test_convert_woodwork_types_wrapper_with_nan():
-    y = _convert_woodwork_types_wrapper(pd.Series([1, 2, None], dtype="Int64"))
-    pd.testing.assert_series_equal(y, pd.Series([1, 2, np.nan], dtype="float64"))
+def test_infer_feature_types():
+    X_dt = pd.DataFrame([[1, 2], [3, 4]])
+    X_dt.ww.init()
+    pd.testing.assert_frame_equal(X_dt, infer_feature_types(X_dt))
 
-    y = _convert_woodwork_types_wrapper(pd.array([1, 2, None], dtype="Int64"))
-    pd.testing.assert_series_equal(y, pd.Series([1, 2, np.nan], dtype="float64"))
+    X_dc = ww.init_series(pd.Series([1, 2, 3, 4]))
```

Comment:
So is this testing that infer_feature_types leaves the data table unchanged?

Reply:
Yea, I'll change the name to `test_infer_feature_types_no_type_change` to make it clearer